test/system/170-run-userns.bats

```diff
@@ -52,3 +52,15 @@ function _require_crun() {
     run_podman 125 run --group-add keep-groups --group-add 457 $IMAGE id
     is "$output" ".*the '--group-add keep-groups' option is not allowed with any other --group-add options" "Check group leaked into container"
 }
+
+@test "podman userns=auto in config file" {
+    skip_if_rootless "the user might not have enough IDs"
+    skip_if_remote "userns=auto is set on the server"
+
+    cat > $tmpdir/userns_auto.conf <<EOF
```

Comment:
`$tmpdir` must be set.  Currently, we're creating a file at `/userns_auto.conf`.
```bash
tmpdir=$PODMAN_TMPDIR/userns-test
mkdir -p $tmpdir                
```

Reply:
...or just use `$PODMAN_TMPDIR` directly

Reply:
thanks, adjusted to use `$PODMAN_TMPDIR`